source/features/parse-backticks.css

```diff
@@ -1,6 +1,6 @@
 /* Style copied from GitHub's <code> */
 .rgh-parse-backticks {
-	background-color: rgba(27, 31, 35, 0.05);
+	background-color: rgb(27 31 35 / 5%);
```

Comment:
If we do this we have to raise the minimum compatible version in manifest.json

Reply:
https://github.com/sindresorhus/refined-github/pull/3168/commits/854e51b524e9d4607d8325b51e7270e263123a79